Xamarin.Essentials/AppInfo/AppInfo.android.cs

```diff
@@ -1,11 +1,22 @@
-﻿using System.Globalization;
+﻿using System;
+using System.Globalization;
+using Android.App;
 using Android.Content;
 using Android.Content.PM;
+using Android.Content.Res;
+using Android.OS;
+using Android.Runtime;
+using static Android.App.Application;
 
 namespace Xamarin.Essentials
 {
     public static partial class AppInfo
     {
+        static readonly Lazy<AppStateLifecycleListener> appState = new Lazy<AppStateLifecycleListener>(
```

Comment:
We actually have an AppStateLifecycleListener that is already used in every application. This should be used.

Reply:
Could you perhaps point me there? I have looked around context, and application and can't seem to find it, sorry.